Week1/Homework/js-exercises/compareArrays.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+var me = ["Abuzer", 29, "unemployed", "bachelor graduate"];
```

Comment:
Could you use `const` or `let` instead of `var`? (if you develop this habbit of not-using var, it will be easier in the future ;))

This is also for the rest of the homework